tests/test_framework/phpbb_functional_test_case.php

```diff
@@ -1140,12 +1140,13 @@ public function create_topic($forum_id, $subject, $message, $additional_form_dat
 	*/
 	public function create_post($forum_id, $topic_id, $subject, $message, $additional_form_data = array(), $expected = '')
 	{
-		$posting_url = "posting.php?mode=reply&f={$forum_id}&t={$topic_id}&sid={$this->sid}";
+		$posting_url = "posting.php?mode=reply&t={$topic_id}&sid={$this->sid}";
 
 		$form_data = array_merge(array(
 			'subject'		=> $subject,
 			'message'		=> $message,
 			'post'			=> true,
+            'topic_id'		=> $topic_id,
```

Comment:
Indentation should be switched from spaces to tabs, to align with coding standard. Same is true for lines 1185 - 1190 added below.